<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>709</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_18">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_info">
       <attribute name="title">
        <string>Application Info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="SeriesInfoWidget" name="widgetSeriesInfo" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_string">
       <attribute name="title">
        <string>String Analysis</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="TabWidgetString" name="stringMainWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_cards">
       <attribute name="title">
        <string>Card Analysis</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="TabWidgetCards" name="cardMainWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menu_new_series">
     <property name="title">
      <string>New Series</string>
     </property>
     <widget class="QMenu" name="menu_file_aircraft">
      <property name="title">
       <string>File Aircraft</string>
      </property>
      <addaction name="action_select_File_Aircraft"/>
      <addaction name="separator"/>
     </widget>
     <addaction name="action_new_series_new_aircraft"/>
     <addaction name="menu_file_aircraft"/>
    </widget>
    <addaction name="menu_new_series"/>
    <addaction name="separator"/>
    <addaction name="action_save"/>
    <addaction name="action_open"/>
    <addaction name="separator"/>
    <addaction name="action_pass_manager"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <widget class="QMenu" name="menuReport_Options">
     <property name="title">
      <string>Report Options</string>
     </property>
     <widget class="QMenu" name="menuCard_Images_per_Page">
      <property name="title">
       <string>Card Images per Page</string>
      </property>
      <addaction name="action5"/>
      <addaction name="action7"/>
      <addaction name="action9"/>
     </widget>
     <widget class="QMenu" name="menuCard_Image_Type">
      <property name="title">
       <string>Card Image Type</string>
      </property>
      <addaction name="actionOriginal"/>
      <addaction name="actionOutline"/>
      <addaction name="actionMask"/>
     </widget>
     <widget class="QMenu" name="menuLogo">
      <property name="title">
       <string>Logo</string>
      </property>
      <addaction name="actionInclude_Logo"/>
      <addaction name="separator"/>
      <addaction name="actionLogo_File"/>
      <addaction name="actionSelect_Logo_File"/>
     </widget>
     <addaction name="actionInclude_Card_Images"/>
     <addaction name="action_downsample_card_images"/>
     <addaction name="menuCard_Image_Type"/>
     <addaction name="menuCard_Images_per_Page"/>
     <addaction name="menuLogo"/>
    </widget>
    <addaction name="actionCreate_Report"/>
    <addaction name="separator"/>
    <addaction name="menuReport_Options"/>
    <addaction name="actionReportManager"/>
   </widget>
   <widget class="QMenu" name="menuExport_to_Excel">
    <property name="title">
     <string>Export</string>
    </property>
    <widget class="QMenu" name="menuOperation_S_A_F_E_Attendee_Log">
     <property name="title">
      <string>Operation S.A.F.E. Attendee Log</string>
     </property>
     <addaction name="action_SAFE_log_from_files"/>
     <addaction name="action_SAFE_log_from_directory"/>
    </widget>
    <addaction name="menuOperation_S_A_F_E_Attendee_Log"/>
    <addaction name="separator"/>
    <addaction name="action_detailed_report"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionUserManual"/>
    <addaction name="actionWRKSpectrometerManual"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <widget class="QMenu" name="menuWorksheets_2">
     <property name="title">
      <string>Worksheets</string>
     </property>
     <addaction name="actionWorksheetWRK"/>
     <addaction name="actionWorksheetGillColor"/>
     <addaction name="actionWorksheetGillBW"/>
    </widget>
    <addaction name="menuWorksheets_2"/>
    <addaction name="separator"/>
    <addaction name="actionCPCatalog"/>
    <addaction name="separator"/>
    <addaction name="actionShortcutStringDrive"/>
    <addaction name="actionShortcutSpectrometer"/>
    <addaction name="separator"/>
    <addaction name="action_reset_defaults"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuExport_to_Excel"/>
   <addaction name="menuReport"/>
   <addaction name="menuExtras"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_Report">
   <property name="text">
    <string>Operation S.A.F.E. Report</string>
   </property>
  </action>
  <action name="action_save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_new_series_new_aircraft">
   <property name="text">
    <string>New Aircraft</string>
   </property>
  </action>
  <action name="action_pass_manager">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pass Manager</string>
   </property>
  </action>
  <action name="action_select_File_Aircraft">
   <property name="text">
    <string>Select File Aircraft</string>
   </property>
  </action>
  <action name="action_detailed_report">
   <property name="text">
    <string>All Raw Data</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUserManual">
   <property name="text">
    <string>AccuPatt User Manual</string>
   </property>
  </action>
  <action name="actionCPCatalog">
   <property name="text">
    <string>CP Nozzles Catalog</string>
   </property>
  </action>
  <action name="actionReportManager">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Report Manager</string>
   </property>
  </action>
  <action name="action_reset_defaults">
   <property name="text">
    <string>Reset All Defaults</string>
   </property>
  </action>
  <action name="actionInclude_Card_Images">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Include Card Images</string>
   </property>
  </action>
  <action name="actionOriginal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </action>
  <action name="actionOutline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Outline</string>
   </property>
  </action>
  <action name="actionMask">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mask</string>
   </property>
  </action>
  <action name="action7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action_downsample_card_images">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Downsample Card Images</string>
   </property>
  </action>
  <action name="actionWRKSpectrometerManual">
   <property name="text">
    <string>WRK Spectrometer Manual</string>
   </property>
  </action>
  <action name="actionInclude_Logo">
   <property name="text">
    <string>Include Logo</string>
   </property>
  </action>
  <action name="actionLogo_File">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Logo File</string>
   </property>
  </action>
  <action name="actionSelect_Logo_File">
   <property name="text">
    <string>Select Logo File</string>
   </property>
  </action>
  <action name="action_SAFE_log_from_files">
   <property name="text">
    <string>From Files</string>
   </property>
  </action>
  <action name="action_SAFE_log_from_directory">
   <property name="text">
    <string>From Directory</string>
   </property>
  </action>
  <action name="actionWorksheetWRK">
   <property name="text">
    <string>WRK</string>
   </property>
  </action>
  <action name="actionWorksheetGillColor">
   <property name="text">
    <string>Gill - Color</string>
   </property>
  </action>
  <action name="actionWorksheetGillBW">
   <property name="text">
    <string>Gill - B&amp;W</string>
   </property>
  </action>
  <action name="actionShortcutStringDrive">
   <property name="text">
    <string>Shortcut - String Drive</string>
   </property>
  </action>
  <action name="actionShortcutSpectrometer">
   <property name="text">
    <string>Shortcut - Spectrometer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SeriesInfoWidget</class>
   <extends>QWidget</extends>
   <header>accupatt/widgets/seriesinfowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabWidgetString</class>
   <extends>QWidget</extends>
   <header>accupatt/widgets/tabWidgetString.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabWidgetCards</class>
   <extends>QWidget</extends>
   <header>accupatt/widgets/tabWidgetCards.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
